### PR TITLE
OPENEUROPA-2088: ECL 2.0: Inpage Navigation.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description
https://ec.europa.eu/component-library/ec/components/navigation/inpage-navigation/usage/

### CODE REVIEW

File: tests/Kernel/Paragraphs/ContentRowTest.php [hunk 1]
```diff
@@ -17,6 +17,7 @@ class ContentRowTest extends ParagraphsTestBase {
    * Tests the rendering of the paragraph type.
    *
    * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+   * @todo Update (restore) after porting to ECL2 other components.
```
Comment (voidtek) [inline tests/Kernel/Paragraphs/ContentRowTest.php]: Can you add the number of the ticket?

Comment (sergepavle) [reply tests/Kernel/Paragraphs/ContentRowTest.php]: Thanks. I have added mentioning of related tickets.

File: templates/paragraphs/paragraph--oe-content-row--variant-inpage_navigation.html.twig [hunk 2]
```diff
@@ -9,7 +9,7 @@
 {{ attach_library('oe_theme/inpage_navigation') }}
 <div class="ecl-row">
```
Comment (voidtek) [inline templates/paragraphs/paragraph--oe-content-row--variant-inpage_navigation.html.twig]: This template doesn't seem the same of https://ecl-twig.netlify.com/ec/?path=/story/components-navigation-inpage-navigation--default

File: tests/Kernel/Paragraphs/ContentRowTest.php [hunk 3]
```diff
@@ -17,6 +17,7 @@ class ContentRowTest extends ParagraphsTestBase {
    * Tests the rendering of the paragraph type.
    *
    * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+   * @todo Update (restore) after porting to ECL2 other components: Link blocks (OPENEUROPA-2092), List item (OPENEUROPA-1989, OPENEUROPA-1990, OPENEUROPA-2124).
    */
   public function testRendering(): void {
```
Comment (voidtek) [inline tests/Kernel/Paragraphs/ContentRowTest.php]: This test is not added on `drone.yml`

File: templates/paragraphs/paragraph--oe-content-row--variant-inpage_navigation.html.twig (hunk 2 above)
Comment (sergepavle) [reply templates/paragraphs/paragraph--oe-content-row--variant-inpage_navigation.html.twig]: I have added wrapper (ecl-container).
Difference still exist but it is related to many aspects of integration. 
For instance we adding id for anchor for div wrapper (not for H1 tag as in example).
Also for many paragraphs we don't usage h1.
But in general inpage navigation currently work without big difference from demo (or I miss something).

File: tests/Kernel/Paragraphs/ContentRowTest.php (hunk 3 above)
Comment (sergepavle) [reply tests/Kernel/Paragraphs/ContentRowTest.php]: Fixed (sorry for that).

**Comment by voidtek** [comment]
thanks for the changes.

**Pull request merged**